pubsub/jetstream/metadata.go

```diff
@@ -20,6 +20,7 @@ import (
 	"time"
 
 	"github.com/dapr/components-contrib/pubsub"
+	"github.com/nats-io/nats.go"
```

Comment:
Your imports need to be rearranged.

there need to be 3 blocks of imports in the following order:

- standard library
- other packages
- dapr packages (github.com/dapr)